src/main/java/frc5124/robot2020/commands/auto/runpos/Turn180.java

```diff
@@ -0,0 +1,43 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc5124.robot2020.commands.auto.runpos;
+
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc5124.robot2020.subsystems.Turret;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class Turn180 extends WaitCommand {
```

Comment:
The command is called turn 180 but it turns to the position -180. That's kind of confusing: consider a rename